test/.eslintrc

```diff
@@ -52,5 +40,20 @@
 		"original$": true,
 		"baseURL": true,
 		"externalHost": true
+	},
+	"rules": {
```

Comment:
Like the new structure, in the sense that we now know what rules failing for `test` folder